mmdet/models/detectors/cascade_rcnn.py

```diff
@@ -109,8 +109,8 @@ def forward_train(self,
                       img,
                       img_meta,
                       gt_bboxes,
+                      gt_bboxes_ignore,
```

Comment:
I suggest keeping the original order and `gt_bboxes_ignore` can be an optional argument.